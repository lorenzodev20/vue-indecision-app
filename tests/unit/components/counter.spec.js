import { shallowMount } from "@vue/test-utils";
import Counter from '@/components/Counter';

describe('Counter Componet', () => {
  test('Debe hacer match con el snapshot',() => {
    
    const wrapper = shallowMount(Counter);

    expect(wrapper.html).toMatchSnapshot()
  })
});