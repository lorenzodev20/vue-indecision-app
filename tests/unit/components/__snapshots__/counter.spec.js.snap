// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Componet Debe hacer match con el snapshot 1`] = `[Function]`;
